package.json

```diff
@@ -49,6 +50,7 @@
     "ember-cli-sri": "^2.1.1",
     "ember-cli-typescript-blueprints": "^2.0.0",
     "ember-cli-uglify": "^3.0.0",
+    "ember-decorators": "^6.0.0",
```

Comment:
Can you `yarn add @ember-decorators/component` instead to same on some bytes? 😄

Reply:
Cool, didn't know that was possible. I just followed the ember-decorators docs which made it sound that the core package was always needed.